### PR TITLE
Fix build on haiku

### PR DESCRIPTION
**Pull request by puckipedia**
As it says on the tin: For GCI task https://google-melange.appspot.com/gci/task/view/google/gci2013/6362387384893440 and Haikuports recipe https://bitbucket.org/puckipedia/haikuports/pull-request/1/add-beindexed-recipe/diff.

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -6,7 +6,7 @@ DIST:=beindexed-$(VERSION)
 SUBDIRS = \
 	src
 
-IMKIT_HEADERS=$(addprefix /boot/home/config/include/, $(wildcard libim/*.h))
+# IMKIT_HEADERS=$(addprefix /boot/home/config/include/, $(wildcard libim/*.h))
```
Comment (waddlesplash) [inline makefile:9]: This leaves IMKit in the code, right? Someday we'll be able to add IMKit, it's still around: https://github.com/HaikuArchives/IMKit

Comment (puckipedia) [inline makefile:9]: It doesn't seem to actually contain any IMKit code: it builds without this line

Comment (waddlesplash) [inline makefile:9]: Ah, OK.

**Comment by waddlesplash** [comment]
Thanks!

File: makefile (hunk 1 above)
Comment (puckipedia) [inline makefile:9]: Also, what use is it to have instant messaging in a searching app?

**Pull request merged**